stdlib/gettext.pyi

```diff
@@ -30,52 +31,83 @@ def find(domain: str, localedir: StrPath | None = ..., languages: Iterable[str]
 
 _T = TypeVar("_T")
 
-@overload
-def translation(
-    domain: str,
-    localedir: StrPath | None = ...,
-    languages: Iterable[str] | None = ...,
-    class_: None = ...,
-    fallback: bool = ...,
-    codeset: str | None = ...,
-) -> NullTranslations: ...
```

Comment:
The `codeset` parameter of both this and the `install` function following it has been removed in 3.11.